README.md

```diff
@@ -1,93 +1,179 @@
+
+    |Type|Protocol|Port Range|Source|
+    |----|--------|----------|------|
+    |SSH|TCP|22|0.0.0.0/0|
+    |SSH|TCP|22|::/0|
+    |HTTP|TCP|80|0.0.0.0/0|
+    |HTTP|TCP|80|::/0|
+    |HTTPS|TCP|443|0.0.0.0/0|
+    |HTTPS|TCP|443|::/0|
+    
+    Finally, click **Review and Launch**.
+9. You should be in the Review Instance Launch page. Check your configurations and click **Launch** to launch the Host VM.
+9. A popup should appear asking you to create your SSH Key Pair. 
+Create a new key pair with a unique name and click on **Download Key Pair** to download it to a CSV file. 
+After confirming your key pairs, the Host VM should be created.
+
+### 2. Generating Elastic IP
+Next, we need to set an Elastic IP for our Host VM. Elastic IPs are just static IPs designed for dynamic cloud computing.
+
+9. Install/Upgrade AWS CLI to the latest version. If you need help installing the AWS CLI,
+please click [here](https://docs.aws.amazon.com/cli/latest/userguide/installing.html).
+9. If you don't know your AWS Access keys and Secret Keys, please go [here](https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html) to setup them up.
+Otherwise, skip to the next step.
+9. Run `aws configure` and input your...
+    - AWS access key
+    - AWS secret key
+    - default region name (The Region of Your Host VM)
+    - default output format (Output format is usually `json`)
+    
+9. Run `aws ec2 allocate-address --domain "vpc" --region <region name>`.
+9. You should receive a text response like the one below.
+    ```json
+    {
+        "PublicIp": "99.99.999.999",
+        "AllocationId": "eipalloc-12345678",
+        "Domain": "vpc"
+    }
+    ```
+    Record the `PublicIP` in this response. This will be used for our Host VM.
+9. Go back to the AWS Console. 
+9. Select the top toolbar. Click **Service**. Then, click **EC2**.
+9. In the sidebar under the **Network & Security** subsection, click the **Elastic IPs** link
+9. In the table, find the Elastic IP you generated earlier.
+9. Click in the **Actions** button on top of the table. Then, click **Associate Address**.
+9. In the Instance textbox, put in the name of your Host VM.
+9. Afterwards, click the caret the **Private IP** textbox. There should be only one ip in the dropdown. That is your Host VMs private IP. Click on that IP.
+9. Finally, click **Associate**.
+
+### 3. Add remaining settings to Security Group
+Now that we created our Elastic IP. We can finish configuring our Security Group. 
+
+9. Add settings in the table below to your own securty group.
+
+|Type|Protocol|Port Range|Source|
+|----|--------|----------|------|
+|ALL TCP|TCP|0 - 65535|\<Elastic IP>|
+|ALL TCP|TCP|0 - 65535|\<Name of your Security Group>|
+
+### 4. Create your Host VM's Subdomain
+Now we need to create a subdomain and link it to for Host VM. Your subdomain should contain your name.
+So if your name is Bob, your subdomain should be `bob.ucsc-cgp-dev.org`.
```

Comment:
Is this really the protocol? What if there's multiple Bobs (or Michaels)?

Reply:
That's a good question. There isn't a strict protocol for subdomain naming and looking at the old documentation, it doesn't say that you can only use your name as the subdomain. Everyone just uses their name. 
So maybe those directions should be something like...

```
Your subdomain should be your name, project, or any name that will help you identify the url.
So if you want to use your name and your name is Bob, your subdomain should be `bob.ucsc-cgp-dev.org`
```